javaslang/src/test/java/javaslang/collection/AbstractSortedSetTest.java

```diff
@@ -9,17 +9,22 @@
 import org.junit.Test;
 
 import java.math.BigDecimal;
+import java.util.Comparator;
+
+import static javaslang.TestComparators.toStringComparator;
 
 public abstract class AbstractSortedSetTest extends AbstractSetTest {
 
     @Override
     abstract protected <T> SortedSet<T> of(T element);
 
+    abstract protected <T> SortedSet<T> of(Comparator<? super T> comparator, T element);
```

Comment:
\[Checkstyle\] ERROR: 'protected' modifier out of order with the JLS suggestions\.